src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -905,7 +904,9 @@ void Subchannel::MaybeStartConnectingLocked() {
 
 void Subchannel::OnRetryAlarm(void* arg, grpc_error* error) {
   Subchannel* c = static_cast<Subchannel*>(arg);
-  gpr_mu_lock(&c->mu_);
+  // TODO(soheilhy): Once subchannel refcounting is simplified, we can get use
+  //                 MutexLock instead of ReleasableMutexLock, here.
+  grpc_core::ReleasableMutexLock lock(&c->mu_);
```

Comment:
This is already in the `grpc_core` namespace, so the prefix should not be needed.

Reply:
Sure, done.